### DISCUSSION TITLE
Facing Issue after upgrading the PVC's

### DISCUSSION THREAD

**Discussion by Lakshay1912** [strimzi/strimzi-kafka-operator#8761] [General]
Hi @scholzj,

I am getting errors in my strimzi kafka cluster can you please help me resolve the issue as soon as possible?
 
here are the logs:

[Archive.zip](https://github.com/strimzi/strimzi-kafka-operator/files/11925497/Archive.zip)

**Comment by scholzj** [comment]
What does _upgrading the PVCs_ mean? And what issue are you facing?
